### DISCUSSION TITLE
Alert configuration

### DISCUSSION THREAD

**Discussion by Mohan-veera** [sni/thruk#1259] [Q&A]
Hello Team,

Is there any possibility of configuring the alert to receive through mail, if the device is "down" and it doesn't comes to "up" state in 20 minutes...?

Currently we are monitoring around 400 devices by using the Thruk and some devices are getting restarted everyday(not on particular time)  and for restarting and getting back to alive they were taking around 10-15 minutes, so the Thruk is sending the ping to those devices during the restart time and if they are not reachable Thruk was sending the mail that device is down. So my query is that is there any possibility to configure the alert to receive through mail (Not Specifically on mail, expecting in API also) when the device is DOWN and it didn't changed the state to UP within 20 minutes...? Can we get the alert in that scenario?

**Comment by Mohan-veera** [comment]
Hello @sni 
Can you please comment on this....? Is there any possibility for the above..

**Reply by sni** [reply]
I'd say this is what `first_notification_delay` is for. Have you tried that option?
